redaxo/src/addons/phpmailer/lib/cronjob.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * PHPMailer Addon.
+ *
+ * @author markus[dot]staab[at]redaxo[dot]de Markus Staab
+ *
+ * @package redaxo\phpmailer
+ */
+
+class rex_cronjob_mailer_purge extends rex_cronjob
+{
+    protected function purgeMailarchive($days = 7, $dir = '', $log = 0)
+    {
+        foreach (glob($dir . '/*') as $file) {
```

Comment:
Wegen Psalm: glob kann ein array oder false zurückliefern.
Schau auch mal was bei den anderen Meldungen "Left operand cannot be mixed" möglich ist.

Reply:
@tbaddade habe jetzt so einiges gemacht. Es sind noch einige Punkte offen, bei denen ich einfach nicht weiter weiß. 
Kannst Du mir noch mal auf die Sprünge helfen? Oder liegt es evtl. eher daran dass psalm aufgrund fehlender Rückgabewerte nicht weiter weiß?